compiler/frontend/src/ccomp/parser_hw/CCompiler.java

```diff
@@ -1847,8 +1847,8 @@ private void builtinTypes() {
 			typetable.put(join(signString, "long long int"),
 					intType(signString, 64)); // "At least 64 bits in size"
 
-			// NONSTANDARD "int128" extension
-			typetable.put(join(signString, "int128"), intType(signString, 128));
+			// NONSTANDARD "field" extension representing the underlying prime field
+			typetable.put(join(signString, "field"), intType(signString, 254));
```

Comment:
It would be awesome if we could deduce the width based on the underlying curve that pepper has been compiled with (this number works for bn128, but it would likely be larger for BLS).

@maxhowald do you have any idea how we could achieve this?